packages/sdks/web-component/test/descope-wc.test.ts

```diff
@@ -71,7 +71,7 @@ const defaultOptionsValues = {
   client: {},
   redirectAuth: undefined,
   tenant: undefined,
-  locale: 'en',
```

Comment:
Note that this changes what is being sent to the server from the browser navigate to also include the dialect